### PR TITLE
VR > Organizer Meta

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/137115
🎥 https://www.loom.com/share/c0dcc3548b6740faa49d88dd28936aa6

- Add icons
- Add variables for icons

### CODE REVIEW

File: variables/_svgs.pcss [hunk 1]
```diff
@@ -222,6 +222,18 @@
...
+/* -----------------------------------------------------------------------------
+ * SVG - Link
+ * ----------------------------------------------------------------------------- */
+
+@svg-load link url('../icons/link.svg') {
+
+	path {
+		stroke: var(--color-icon-active);
+	}
+
```
Comment (paulmskim) [inline variables/_svgs.pcss]:
```suggestion
```

File: variables/_svgs.pcss [hunk 2]
```diff
@@ -265,6 +277,18 @@
...
+/* -----------------------------------------------------------------------------
+ * SVG - Mail
+ * ----------------------------------------------------------------------------- */
+
+@svg-load mail url('../icons/mail.svg') {
+
+	path {
+		stroke: var(--color-icon-active);
+	}
+
```
Comment (paulmskim) [inline variables/_svgs.pcss]:
```suggestion
```

File: variables/_svgs.pcss [hunk 3]
```diff
@@ -299,6 +323,18 @@
...
+/* -----------------------------------------------------------------------------
+ * SVG - Phone
+ * ----------------------------------------------------------------------------- */
+
+@svg-load phone url('../icons/phone.svg') {
+
+	path {
+		stroke: var(--color-icon-active);
+	}
+
```
Comment (paulmskim) [inline variables/_svgs.pcss]:
```suggestion
```

File: variables/_svgs.pcss [hunk 4]
```diff
@@ -222,6 +222,18 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Link
+ * ----------------------------------------------------------------------------- */
+
+@svg-load link url('../icons/link.svg') {
```
Comment (paulmskim) [inline variables/_svgs.pcss]: Let's follow color variable names in the svg var name, much like what we've done previously. So we should call this `link-active`. Same with `mail` and `phone` below.

File: variables/_svgs.pcss (hunk 1 above)
Comment (juanfra) [reply variables/_svgs.pcss]: The rest of the variables have this trailing newline. If I remove the line will be inconsistent.

File: variables/_svgs.pcss (hunk 2 above)
Comment (juanfra) [reply variables/_svgs.pcss]: The rest of the variables have this trailing newline. If I remove the line will be inconsistent.

File: variables/_svgs.pcss (hunk 3 above)
Comment (juanfra) [reply variables/_svgs.pcss]: The rest of the variables have this trailing newline. If I remove the line will be inconsistent.

File: variables/_svgs.pcss (hunk 1 above)
Comment (bordoni) [reply variables/_svgs.pcss]: No need to change. Keep it consistent

File: variables/_svgs.pcss (hunk 2 above)
Comment (bordoni) [reply variables/_svgs.pcss]: No need to change. Keep it consistent

File: variables/_svgs.pcss (hunk 3 above)
Comment (bordoni) [reply variables/_svgs.pcss]: No need to change. Keep it consistent

Comment (bordoni) [reply variables/_svgs.pcss]: I took another look, the file doesnt have the last spacing. So I am pushign the suggestion from @paulmskim

File: variables/_svgs.pcss (hunk 2 above)
Comment (bordoni) [reply variables/_svgs.pcss]: I took another look, the file doesnt have the last spacing. So I am pushign the suggestion from @paulmskim

File: variables/_svgs.pcss (hunk 1 above)
Comment (bordoni) [reply variables/_svgs.pcss]: I took another look, the file doesnt have the last spacing. So I am pushign the suggestion from @paulmskim

**Pull request merged**